.docker/r-api

```diff
@@ -0,0 +1,21 @@
+FROM rocker/r-ver:4.0.2
+
+WORKDIR /usr/local/optisel
+
+RUN apt update \
+  && apt install -y --no-install-recommends \
+  xvfb \
+  libpq-dev \
+  libx11-dev \
+  pkg-config \
+  libglu1-mesa-dev \
+  zlib1g-dev \
+  libxml2-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY . .
+
+RUN MAKEFLAGS='-j4' Rscript -e 'install.packages(c("here","RPostgres","optiSel","plumber","jsonlite","devtools"),clean=TRUE, Ncpus=8);\
+                    library(devtools);devtools::install_github("rmsharp/nprcgenekeepr")'
```

Comment:
Do we want to lock these to a specific version or is that not necessary?

Reply:
Yes perhaps that is a good Idea at least R version can completely destroy this so that should be fixed version